### PR TITLE
Update to the topic 2.1

### PR DESCRIPTION
**Pull request by Gowtham-GitHub**
1.Web Identity Federation modified
2.SCP explained in detail

### CODE REVIEW

File: saa/2.1. Security - IAM & Organizations.md [hunk 1]
```diff
@@ -87,7 +87,7 @@
 ## AWS Organizations
   
 - Consolidate multiple AWS accounts into an organization that you create and centrally manage
-  - Enforce (SCP) policies centrally on root or OU's.
+  - Enforce Service Control Policy(SCP) centrally on root or OU's. Also SCP has precedence over individual's permission. Best suitable for restricting access across multiple accounts in an organization.
```
Comment (undergroundwires) [inline saa/2.1. Security - IAM & Organizations.md]: Single whitespace before parentheses 😀

File: saa/2.1. Security - IAM & Organizations.md [hunk 2]
```diff
@@ -19,8 +19,8 @@
 - Identity Federation
   - Federating Users of Mobile/Web app using **Cognito**
     - Using Amazon Cognito will create identities(with the help of Mobile SDK) for users and sync across devices.
-  - Federating Users with **OpenID Connect** or public identity service provider
-    - Using third-party services like Facebook, Google or any identity providers(IdP) compatible with OIDC can federate users. But Amazon recommends using Cognito.
+  - Federating Users with **Web Identity Federation** OpenID Connect(OIDC) or public identity service provider
```
Comment (undergroundwires) [inline saa/2.1. Security - IAM & Organizations.md]: Single whitespace before parentheses 😀

File: saa/2.1. Security - IAM & Organizations.md (hunk 1 above)
Comment (Gowtham-GitHub) [reply saa/2.1. Security - IAM & Organizations.md]: Thanks for picking that up.

File: saa/2.1. Security - IAM & Organizations.md (hunk 2 above)
Comment (Gowtham-GitHub) [reply saa/2.1. Security - IAM & Organizations.md]: That one will do fine.

**Pull request merged**